### ISSUE TITLE
Check Doesn't Work

### ISSUE THREAD

**Issue by bearrito** [ansiblebit/oracle-java#16]
Version: 1.9.4

Unable to run in check mode.  Maybe that is a reasonable limitation, but it's a bit of a workaround to deal with.

```
TASK: [ansiblebit.oracle-java | echo oracle_java_task_installed] ************** 
<machine> ESTABLISH CONNECTION FOR USER: user
ok: [machine] => {
    "msg": "oracle_java_task_installed={'msg': u'check mode not supported for shell', 'invocation': {'module_name': u'shell', 'module_complex_args': {}, 'module_args': u'which java'}, 'skipped': True}"
}

TASK: [ansiblebit.oracle-java | set fact oracle_java_installed] *************** 
<machine> ESTABLISH CONNECTION FOR USER: user
fatal: [machine] => One or more undefined variables: 'dict object' has no attribute 'rc'

FATAL: all hosts have already failed -- aborting
```

**Comment by steenzout** [comment]
@bearrito please try v4.9.8.

**Comment by steenzout** [comment]
fix included in #20 #21
